internal/deviations/deviations.go

```diff
@@ -658,4 +677,8 @@ var (
 	interfaceRefConfigUnsupported = flag.Bool("deviation_interface_ref_config_unsupported", false, "Device does not support interface-ref configuration when applying features to interface")
 
 	storageComponentUnsupported = flag.Bool("deviation_storage_component_unsupported", false, "Set to true for device that does not support telemetry path /components/component/storage")
+
+	_ = flag.Bool("deviation_platform_power_down_up_skip", false, "Set flag to skip as Device does not support power-admin-state leaf")
+
+	_ = flag.Int64("deviation_platform_power_enable_wait", 0, "Set timer in minutes to wait for shut platform to power up")
```

Comment:
These lines can be removed, since these are new deviations, they do not have to be introduced as flags.